### DISCUSSION TITLE
2023/03/09/create-teams-bot-hands-on/

### DISCUSSION THREAD

**Discussion by giscus** [cloudmatelabs/tech.cloudmt.co.kr#150] [Blog Comments]
# 2023/03/09/create-teams-bot-hands-on/

팀즈는 마이크로소프트에서 제공하는 기업 및 학교를 위한 메신저 프로그램입니다. 기본적인 메신저 기능뿐만 아니라 화상회의, PC 원격 조작, 파일 공유 등의 다양한 기

https://tech.cloudmt.co.kr/2023/03/09/create-teams-bot-hands-on/

**Comment by yo-young** [comment]
안녕하세요 팀즈 봇을 이용해서 sharepoint에 공유되고 있는 엑셀 파일을 수정하는 것도 가능할까요?

**Reply by AhatLi** [reply]
죄송합니다. 제가 너무 늦게 봤네요... 이미 해결 하셨겠지만은 답글을 달자면은 제가 알기로는 Graph API를 이용해서 SharePoint에 공유되고 있는 엑셀파일을 수정하는것이 가능한 것으로 알고 있습니다. 봇 내부에서 해당 API를 사용하시면 수정이 가능할 것 같습니다.
관련 문서는 아래와 같습니다.
https://learn.microsoft.com/en-us/graph/api/resources/excel?view=graph-rest-1.0

**Reply by AhatLi** [reply]
만약 저런 특정 셀의 값을 단순히 바꾸는 정도의 기능이 아닌 엑셀 화면을 보는 것 처럼 구현이 가능한지 물으신거라면 봇 앱으로는 불가능 할 것 같습니다.
그런 경우에는 팀즈 탭 앱을 사용해서 구현하여야 합니다.

**Reply by yo-young** [reply]
회사에서 재택여부를 엑셀로 관리하고 있는데 봇으로 업데이트 가능할까 해서 문의드렸습니다. 해당 API 보고 한번 구현해봐야겠네요. 감사합니다.
